src/components/ActionDot.icon.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
No, it is because the parent PR doesn't have the icons, but the saga effect to load the nano contract history. The only way to solve this puzzle is by merging the parent PR.

Reply:
I don't understand, the component PR you linked here also doesn't have the icons, how would it solve this?

Reply:
Nevermind, re-checked the code and the icons were moved